lib/matplotlib/type1font.py

```diff
@@ -309,7 +309,7 @@ def suppress(tokens):
 
         while True:
             token, value = next(tokens)
-            if token == 'name' and value in table:
+            if token is cls._name and value in table:
```

Comment:
Why use `is` test instead of `==`?  That seems to be betting heavily on some cpython internals working as expected.